lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -150,7 +154,8 @@ export default function AssignmentActivity() {
 
     return students.data.students
       .filter(
-        ({ auto_grading_grade }) =>
+        ({ auto_grading_grade, active }) =>
+          active &&
```

Comment:
Inactive users should not sync their grades.

Reply:
Ah, it does look simple, nice :+1: